src/Sulu/Bundle/ContactBundle/Entity/AccountRepository.php

```diff
@@ -188,7 +188,8 @@ public function findByIds($ids)
             ->addSelect('faxType')
             ->addSelect('notes')
             ->addSelect('medias')
-            ->where('account.id IN (:accountIds)');
+            ->where('account.id IN (:accountIds)')
+            ->orderBy('account.id', 'ASC');
```

Comment:
For the accounts only its probably feasible. For the whole Sulu project it's a lot of work.